resources/translations/en.json

```
@@ -361,5 +361,10 @@
         "description": "We have sent you a welcome email."
       }
     }
+  },
+  "error": {
+    "title": "Oops! Something went Wrong.",
+    "description": "We're sorry, but there seems to be a problem on our end. Our team has been notified, and we're working to fix this as quickly as possible. Please try refreshing the page in a few minutes. If the problem persists, feel free to contact us at technical-support@publiq.be for assistance. We appreciate your understanding and patience.",
```

Maybe you could change to email into a link with a `mailto:` ?
```suggestion
    "description": "We're sorry, but there seems to be a problem on our end. Our team has been notified, and we're working to fix this as quickly as possible. Please try refreshing the page in a few minutes. If the problem persists, feel free to contact us at <a href="mailto:technical-support@publiq.be">technical-support@publiq.be</a> for assistance. We appreciate your understanding and patience.",
```